cmd/prepare-storage.go

```diff
@@ -134,22 +134,21 @@ func quickErrToActions(errMap map[error]int) InitActions {
 // - Unformatted setup
 //   - Format/Wait for format when `disksUnformatted == diskCount`
 //
-// - Wait for all when `disksUnformatted + disksOffline == disksCount`
+// - Wait for all when `disksUnformatted + disksOffline + disksUnformatted == disksCount`
```

Comment:
the comment seems odd. `disksUnformatted` appears twice in the expression above.

Reply:
Done